scripts/beachball/utils.ts

```diff
@@ -0,0 +1,49 @@
+import { AllPackageInfo, getAllPackageInfo, isConvergedPackage } from '../monorepo';
+
+/**
+ * Reads package info from the monorepo and generates the scopes for beachball bump and release.
+ * Differentiates between vNext and v8 releases.
+ *
+ * vNext scope includes all packages that have version > 8.x and shared internal packages that need versions bumped.
+ * @returns {string[]} Array of package paths for beachball scope
+ */
+export function getConfig({
+  version,
+  allPackageInfo = getAllPackageInfo(),
+}: {
+  version: 'v8' | 'vNext';
+  allPackageInfo?: AllPackageInfo;
+}) {
+  const vNextPackagePaths = getVNextPackagePaths(allPackageInfo);
```

Comment:
```suggestion
export function getConfig({
  version,
}: {
  version: 'v8' | 'vNext';
}) {
  const allPackageInfo = getAllPackageInfo()
  const vNextPackagePaths = getVNextPackagePaths(allPackageInfo);
```

nothing really uses this so I would say to remove it until it's necessary. Realistically the only way this will be used is the result of `getAllPackageInfo`